composer.json

```diff
@@ -21,29 +21,33 @@
     ],
     "require": {
         "php": ">=7.4",
+        "composer/composer": "^2.1",
         "composer/installers": "^1.7",
-        "cweagans/composer-patches": "^1.7",
-        "drupal-composer/preserve-paths": "^0.1.6",
```

Comment:
Is this no longer needed to keep the custom modulea and themes dirs from getting deleted?

Reply:
Thanks for catching that, I was comparing with https://github.com/UN-OCHA/pocam8-site/pull/78/files, but preserve-paths wasn't included until https://github.com/UN-OCHA/pocam8-site/pull/81/files - not sure if it's necessary (and if so, how it was working before), but it's included now, and all builds fine.